src/main/webapp/resources/js/pages/projects/project-analyses/components/AnalysesNav.jsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "@reach/router";
+import { Menu } from "antd";
+import React from "react";
+
+/**
+ * Component to handle navigation within the project analyses page
+ * @param {string} path - the current page name
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export default function AnalysesNav({ path }) {
+  const [key, setKey] = React.useState();
+
+  React.useEffect(() => {
+    setKey(path.split("/")[0]);
```

Comment:
Check to ensure you actually need to do this.  I that the path has what you are looking for.

Reply:
This file has been removed as its no longer required